framework/components/ecl-tabs/ecl-tabs.twig

```diff
@@ -0,0 +1,20 @@
+<div class="ecl-tabs">
```

Comment:
Absolutely. The template has to be rewritten to introduce some logic into it. So does a couple of other components. We can discuss that tomorrow :)